common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1139,6 +1139,10 @@ public final class LC {
     public static final KnownKey smtp_to_lmtp_enabled = KnownKey.newKey(false);
     public static final KnownKey smtp_to_lmtp_port = KnownKey.newKey(7024);
 
+    @Supported
+    public static final KnownKey modify_prefs_logging_value_enabled = KnownKey.newKey(false);
+    public static final KnownKey modify_prefs_delimiter = KnownKey.newKey(";");
```

Comment:
need LC attribute for limiting the number of characters to get logged for the changed prefs setting value

Reply:
Done